r_basics_introduction/r_basics_introduction.md

```diff
@@ -0,0 +1,784 @@
+
+<div style="display:none">@gifPreload</div>
+
+<figure>
+  <img src="https://github.com/arcus/education_modules/blob/r_basics_introduction/r_basics_introduction/media/viewer.png?raw=true" height="540" width="878" alt="RStudio can create a new project that gets its contents from a git repository." data-alt="https://github.com/arcus/education_modules/blob/r_basics_introduction/r_basics_introduction/media/viewer.gif?raw=true" style = "border: 1px solid rgb(var(--color-highlight));">
+
+<figcaption style = "font-size: 1em;">Click on the image to play the demo of opening a viewer of a data frame.</figcaption>
+</figure>
+
+Here are the answers to the questions posed in `introduction_exercises.Rmd`:
+
+1. How many rows are in the data frame? How many columns?
+
+There are 15,524 rows and 17 columns. This information can be seen at the bottom of the Data Viewer, and also in the Environment pane. Note that in R parlance, rows are observations, and columns are variables.
+
+2. Go ahead and try to edit one of the values in this viewer. You will find that you can't. It would have been easy for the RStudio programmers to allow editing of specific values, but they decided not to add that feature. Why do you think this was designed that way?
+
+This would encourage point-and-click workflows that are not reproducible.
+
+3. Each row in *covid_testing* represents a single COVID19 test. The value in the *pan_day* column indicates the day (after the start of the pandemic) on which the test sample was taken. On which "pandemic day" was the first test sample taken? The last?
+
+If you sort the table by `pan_day` and then look at the very top and the very bottom, you will find out the first "pandemic day" was 4, and the last one was 107. You can sort the data in the Data Viewer by clicking on the column header.
+
+Important: this doesn't actually change the data in the `covid_testing` data frame!
+
+4. How many tests overall were positive? How many positive tests were there in the first 30 days of the pandemic? (Hint: look at the Filter button)
+
+The `result` column holds that information. We can apply a filter to that column to only show us rows where the value in the `result` column is "positive". Click on Filter and enter "positive" in the text field that appears below the `result` header. You will now see that the bottom line of the data viewer reads "Showing 1 to [some number] of 865 entries, 17 total columns (filtered from 15,524 total entries)". So there were 865 positive tests, out of 15,524 total.
+
+## Recap
+
+Let's recap what we have covered in this session.
+
+![R logo](media/r_logo.png)<!-- style="max-width:300px; margin:2em auto;" --> ![RStudio logo](media/rstudio_logo.png)<!-- style="max-width:300px; margin:2em auto;" -->![R Markdown logo](media/r__markdown_logo.png)<!-- style="max-width:300px; margin:2em auto;" -->
```

Comment:
these images are rendering strangely (and RMarkdown isn't showing up at all)

Reply:
I can't get things to show up small and side by side, it's super annoying.  I'm just taking these out, as they don't really add anything.